sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleHistogramAggregator.java

```diff
@@ -58,7 +65,8 @@ public final HistogramAccumulation merge(HistogramAccumulation x, HistogramAccum
     for (int i = 0; i < x.getCounts().length; ++i) {
       mergedCounts[i] = x.getCounts()[i] + y.getCounts()[i];
     }
-    return HistogramAccumulation.create(x.getSum() + y.getSum(), mergedCounts);
+    // Note: we always preserve the left-side exemplars as these are the "newer" ones.
```

Comment:
This seems to be the opposite of the CountAggregator choice.

Reply:
Yeah, I wasn't paying close enough attention to my backport.  I had flipped the order when reworking the Aggregator interface, but haven't driven that change here yet.  Fixed.